fsButler/utils.py

```diff
@@ -9,14 +10,15 @@
 import lsst.analysis.utils as utils
 import lsst.afw.display.ds9 as ds9
 
-from lsst.pex.exceptions import LsstCppException
-from lsst.meas.extensions.multiShapelet import FitPsfModel
+# from lsst.pex.exceptions import LsstCppException
```

Comment:
(Here and throughout) please don't comment out code and leave it in the file. If it's not needed, remove it.